robot/Cumulus/tests/browser/npsp_settings/verify_payment_allocations_setting.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Verify Payment Allocations is Disabled by Default
+    [tags]  unstable
+    Populate Field By Placeholder    Quick Find    Custom Settings
```

Comment:
Question: There's a direct link available to the Custom Settings home, `/lightning/setup/CustomSettings/home`. Would it be more stable to navigate via relative URL rather than manipulating Quick Find?

Reply:
changed